### PR TITLE
Deprecate function value for width and height

### PR DESCRIPTION
**Pull request by MikeVitik**

### CODE REVIEW

**Review by churkin — DISMISSED** [review]

File: js/core/dom_component.js [hunk 1]
```diff
@@ -58,11 +58,23 @@ const DOMComponent = Component.inherit({
...
     },
 
+    _checkFunctionValueDeprecation: function(optionNames) {
+        if(!this.option('_ignoreFunctionValueDeprecation')) {
+            optionNames.forEach(optionName => {
+                if(isFunction(this.option(optionName))) {
+                    errors.log('W0017', optionName);
+                }
+            });
+
```
Comment (ksercs) [inline js/core/dom_component.js]:
```suggestion

```

File: js/ui/lookup.js [hunk 2]
```diff
@@ -689,7 +690,7 @@ const Lookup = DropDownList.inherit({
             closeOnTargetScroll: false,
             onPositioned: null,
 
-            maxHeight: function() { return $(window).height(); },
+            maxHeight: '100hv',
```
Comment (ksercs) [inline js/ui/lookup.js]: ?
```suggestion
            maxHeight: '100vh',
```

**Review by ksercs — COMMENTED** [review]
What about using this options in inheritors? 

Try to search by `this._getOptionValue('width',` or something like this

**Review by andrewmakarov — DISMISSED** [review]

**Review by ksercs — DISMISSED** [review]

**Comment by ksercs** [comment]
Shouldn't we also change overlay default width and height?

**Review by ksercs — APPROVED** [review]

**Pull request merged**